src/AbstractActionHandler.test.ts

```diff
@@ -1,4 +1,4 @@
-import { Block, IndexState } from "../../../index"
+import { Block, IndexState } from "./interfaces"
```

Comment:
The build failed because the linting rules say this should be below the next line.